packages/truffle-debugger/lib/evm/sagas/index.js

```diff
@@ -27,11 +27,13 @@ export function *addContext(contractName, binary) {
  */
 export function *addInstance(address, binary) {
   let search = yield select(evm.info.binaries.search);
-  let { context } = search(binary);
+  if (binary != "0x0") {
+    let { context } = search(binary);
 
-  yield put(actions.addInstance(address, context, binary));
+    yield put(actions.addInstance(address, context, binary));
 
-  return context;
+    return context;
```

Comment:
Surely you need to return `context` regardless of if an instance is emitted?

This will break method chaining if it returns `null`

Reply:
I don't think those return values are actually used... I think returning `undefined` is fine.